### ISSUE TITLE
Go to the equivalent page when switching versions, if possible

### ISSUE THREAD

**Issue by hdgarrood** [purescript/pursuit#103]
If I am on the page for the Data.Maybe module version 0.3.2, and I use the version selector to select 0.3.1, then I should be taken to the Data.Maybe module for 0.3.1, not the package homepage for 0.3.1.

If that module does not exist, then I think it's ok to show a 404 page, as long as it explains what happened.

**Comment by michaelficarra** [comment]
I'm not sure if that's better than the current experience. At least with the way things are now, I never get a 404.

**Comment by hdgarrood** [comment]
Fair enough - how do you feel about making it so that that a request for a non-existent module in an existing package will redirect to the package homepage for that version?

eg: `/packages/purescript-maybe/0.3.1/docs/Does.Not.Exist` => `/packages/purescript-maybe/0.3.1/`

**Comment by michaelficarra** [comment]
Yeah, that works.
